src/main/webapp/requestResendLinkSuccess.jsp

```diff
@@ -0,0 +1,10 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<t:staticPage>
+  <h1 class="color-orange">
+    Request for Resend of Feedback Links
+  </h1>
+  <p>Request Success! An email containing links of feedback sessions which you have participated in the recent one month is sent to your email.</p>
```

Comment:
"... containing links to all feedback sessions which you participated in the last one month..."